docs/book/themes/themes.rst

```diff
@@ -0,0 +1,110 @@
+.. index::
+   single: Themes
+
+Themes
+======
+
+Theming is method of customizing how do your channels look like in Sylius. Each channel can have a different Theme.
```

Comment:
Theming is **a** method [...] how ~do~ your channels [...]

Theme is capitalized at the end, but lowercased in the rest of the text